lib/config/presets/internal/group.ts

```diff
@@ -265,7 +265,7 @@ const staticGroups = {
     packageRules: [
       {
         groupName: 'spring boot',
-        matchPackagePrefixes: ['org.springframework.boot:'],
+        matchPackagePrefixes: ['org.springframework.boot:', 'org.springframework.boot'],
```

Comment:
This won't work as you intended. All prefixes are matched with `startsWith`

Reply:
Thanks @viceice for the information.
So, it's not considering a pattern as well, something like `'^org.springframework.boot$'` won't work then.
Would a mix of `matchPackagePrefixes` and  `matchPackageNames` work? Like:

```
"packageRules": [
    {
      "groupName": "spring boot",
      "matchPackagePrefixes": [
        "org.springframework.boot:"
      ],
      "matchPackageNames": [
        "org.springframework.boot"
      ]
    }
  ]
```

Reply:
It does the trick it seems!

```
DEBUG: Using group branchName template (repository=CAPI/my-project)
DEBUG: Dependency org.springframework.boot:spring-boot-dependencies is part of group spring boot (repository=CAPI/my-project)
DEBUG: Using group branchName template (repository=CAPI/my-project)
DEBUG: Dependency org.springframework.boot:spring-boot-devtools is part of group spring boot (repository=CAPI/my-project)
DEBUG: Using group branchName template (repository=CAPI/my-project)
DEBUG: Dependency org.springframework.boot:spring-boot-dependencies is part of group spring boot (repository=CAPI/my-project)
DEBUG: Using group branchName template (repository=CAPI/my-project)
DEBUG: Dependency org.springframework.boot is part of group spring boot (repository=CAPI/my-project)
```